src/Conversion/ONNXToKrnl/Math/DFT.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===--------------- DFT.cpp - Lowering DFT Op ----------------===//
+//
+// Copyright 2024 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file lowers the ONNX DFT Operator to Krnl dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp"
+#include "src/Dialect/ONNX/ONNXOps/ShapeHelper.hpp"
+
+// Lowering Computation for DFT is incomplete !!
+
+using namespace mlir;
+
+namespace onnx_mlir {
+
+template <typename OP>
+constexpr bool isAxisInput = std::is_same_v<OP, ONNXDFTOp>;
+
+template <typename OP_TYPE, typename OP_ADAPTOR>
+LogicalResult ONNXDFTOpLoweringCommon(OP_TYPE dftOp, OP_ADAPTOR adaptor,
+    ConversionPatternRewriter &rewriter, const TypeConverter *typeConverter) {
+  Operation *op = dftOp.getOperation();
+  Location loc = ONNXLoc<OP_TYPE>(op);
+  ValueRange operands = adaptor.getOperands();
+
+  //   using MDBuilder =
+  //       MultiDialectBuilder<KrnlBuilder, IndexExprBuilderForKrnl,
+  //       MathBuilder,
+  //           MemRefBuilder, VectorBuilder, AffineBuilderKrnlMem, SCFBuilder>;
+
+  Value input = adaptor.getInput();
+
```

Comment:
Since the lowering of DFT is not supported, add the explicit error return here.

Reply:
Okay I will add it